packages/host/app/components/card-catalog-modal.gts

```diff
@@ -242,6 +242,7 @@ export default class CardCatalogModal extends Component<Signature> {
       .more-actions > svg {
         height: 100%;
       }
+
```

Comment:
typo?

Reply:
No that’s the new template css prettier. It wants to add an extra new line at the end.

Reply:
Oh no, I think that's a bug. In the [original PR](https://github.com/prettier/prettier/pull/15087) that works fine (Prettier 3)...

But when we [backported it](https://github.com/ef4/prettier/commits/8c4a6ad) to Prettier 2 (because we discovered [ember-template-lint-plugin-prettier](https://github.com/ember-template-lint/ember-template-lint-plugin-prettier) uses 2 instead of 3) it looks like this new line bug appeared.

I'll note to fix it in a ticket but until then I guess we'll have to stomach this extra new line annoyance. I have no idea how we missed that when testing 😢